example/ditto/config.yml

```diff
@@ -0,0 +1,4 @@
+components: true
+projects:
+  - name: '[Demo] Onboarding Flow'
+    id: 61e72388365c930170607378
```

Comment:
Did this get accidentally added? I think this should be in `example/src/ditto` instead.